deferred-iterator.js

```diff
@@ -2,39 +2,84 @@
 
 const AbstractIterator = require('abstract-leveldown').AbstractIterator
 const inherits = require('inherits')
+const getCallback = require('./util').getCallback
+
+const kOptions = Symbol('options')
+const kIterator = Symbol('iterator')
+const kOperations = Symbol('operations')
+const kPromise = Symbol('promise')
 
 function DeferredIterator (db, options) {
   AbstractIterator.call(this, db)
 
-  this._options = options
-  this._iterator = null
-  this._operations = []
+  this[kOptions] = options
```

Comment:
So these become private since the symbols are only accessible from this file?

Reply:
Aye.